ObjectContainer/src/ro/sciit/comparators/PersoaneAgeComparator.java

```diff
@@ -0,0 +1,20 @@
+package ro.sciit.comparators;
+import ro.sciit.entities.Persoane;
+import java.util.Comparator;
+
+/**
+ * @author Georgiana
+ *compares two persons by age
+ */
+
+public class PersoaneAgeComparator implements Comparator<Persoane> {
+
+    @Override
+    public int compare(Persoane p1, Persoane p2) {
+        if(p1.getAge() > p2.getAge()){
+            return 1;
+        } else {
+            return -1;
+        }
```

Comment:
What if the ages are identical? :)
Another shorter solution would be: `return p1.getAge() - p2.getAge();`